test/ImageTest.php

```diff
@@ -35,14 +36,55 @@ public function testImageDataProvider() {
         );
     }
 
+    public function testImageDataWithoutAltProvider() {
+        $this->expectNotToPerformAssertions();
```

Comment:
Remove this `expectNotToPerformAssertions` call and simply remove the `test` prefix to the method name.